.github/workflows/luajit-integration.yml

```diff
@@ -0,0 +1,79 @@
+name: LuaJIT integration testing
+
+on:
+  workflow_call:
+    inputs:
+      buildtype:
+        description: CMake build type value
+        required: false
+        type: string
+        default: Debug
+      GC64:
+        description: CMake option value to enable LuaJIT GC64 mode
+        required: false
+        type: string
+        default: OFF
+      host:
+        description: Type of machine to run the GitHub job on
+        required: true
+        type: string
+      release:
+        description: Git revision from tarantool/tarantool repository
+        required: false
+        type: string
+        default: master
+      revision:
+        description: Git revision from tarantool/luajit repository
+        required: true
+        type: string
+
+jobs:
+  luajit-integration:
+    runs-on: ${{ inputs.host }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          repository: tarantool/tarantool
+          fetch-depth: 0
+          submodules: recursive
+          ref: ${{ inputs.release }}
+      - name: LuaJIT bump
+        run: |
+          pushd third_party/luajit
+          git fetch origin
+          git checkout ${{ inputs.revision }}
+          popd
+          # XXX: export the following environment variables to
+          # suppress git config warning. The commit is required to
+          # avoid resetting dirty changes in LuaJIT submodule.
+          export GIT_COMMITTER_NAME="LuaJIT integration testing workflow"
+          export GIT_COMMITTER_EMAIL="noreply@tarantool.org"
+          export GIT_AUTHOR_NAME="LuaJIT integration testing workflow"
+          export GIT_AUTHOR_EMAIL="noreply@tarantool.org"
+          git commit -m "luajit: integration testing bump" third_party/luajit
+        shell: bash
```

Comment:
Seems we can omit it.

Reply:
Some artefacts left after the first attempt (that was using Actions mechanism).